test-it/karma/utilities/TestHelpers.VireoRunner.js

```diff
@@ -3,10 +3,14 @@
     window.testHelpers = window.testHelpers || {};
 
     var rebootAndLoadVia = function (vireo, viaAbsolutePath) {
-        expect(viaAbsolutePath).toBeNonEmptyString();
+        // Jasmine Matchers library is not always ready in beforeAll so use jasmine core functions
+        expect(typeof viaAbsolutePath).toBe('string');
+        expect(viaAbsolutePath).not.toBe('');
```

Comment:
Oh. I misunderstood.

If you read the comment it says jasmine matchers are not available yet. So we can't use toBeNonEmptyString in this file